src/python/install.sh

```diff
@@ -304,7 +304,7 @@ sudo_if() {
     if [ "$(id -u)" -eq 0 ] && [ "$USERNAME" != "root" ]; then
         su - "$USERNAME" -c "$COMMAND"
     else
-        "$COMMAND"
+        $COMMAND
```

Comment:
Can we bump the minor version? Thanks! https://github.com/devcontainers/features/blob/1f83ebc28c0b7966726151d8712c785c5c5f4f99/src/python/devcontainer-feature.json#L3

Reply:
Fixed in f4ff7f9920ea62151fc90d9342e7a128042da4fd